### PR TITLE
[Transforms] Update baselines from merging with master round 2

### PR DESCRIPTION
**Pull request by yuit**
@rbuckton @vladima @mhegazy @sandersn

### CODE REVIEW

**Comment by sandersn** [comment]
:+1:

File: tests/baselines/reference/decoratorOnClass5.es6.js [hunk 1]
```diff
@@ -16,10 +16,10 @@ var __decorate = (this && this.__decorate) || function (decorators, target, key,
     else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
     return c > 3 && r && Object.defineProperty(target, key, r), r;
 };
-let C_1 = class C {
+let C_1;
```
Comment (rbuckton) [inline tests/baselines/reference/decoratorOnClass5.es6.js]: This is wrong. This was fixed by #8786 in master and indicates a change we also need to make here.

File: tests/baselines/reference/decoratorOnClass6.es6.js [hunk 2]
```diff
@@ -16,10 +16,10 @@ var __decorate = (this && this.__decorate) || function (decorators, target, key,
     else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
     return c > 3 && r && Object.defineProperty(target, key, r), r;
 };
-let C_1 = class C {
+let C_1;
```
Comment (rbuckton) [inline tests/baselines/reference/decoratorOnClass6.es6.js]: Also incorrect. I would leave the existing baseline.

File: tests/baselines/reference/decoratorOnClass7.es6.js [hunk 3]
```diff
@@ -16,10 +16,10 @@ var __decorate = (this && this.__decorate) || function (decorators, target, key,
     else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
     return c > 3 && r && Object.defineProperty(target, key, r), r;
 };
-let C_1 = class C {
+let C_1;
```
Comment (rbuckton) [inline tests/baselines/reference/decoratorOnClass7.es6.js]: Also incorrect. I would leave the existing baseline.

**Comment by yuit** [comment]
@rbuckton Gotcha. I will get the fix in

**Pull request merged**